isodesign/io.py

```diff
@@ -20,8 +20,56 @@
 
 """
 
+import pandas as pd 
+from pathlib import Path
 
+class IoGestion:
+    def __init__(self):
+        self.netw = None
+        self.tvar = None
+        self.mflux = None
 
+    def read_file(self, data : str):
+        try:
+            if isinstance(data, str):
+                data_path = Path(data).resolve()
+                if data_path.exists(): 
+                    if data_path.suffix in [".netw"]:
+                        self.netw = pd.read_csv(data, sep='\t', skiprows=[0], header=None)
+                    elif data_path.suffix in [".tvar"]:
+                        self.tvar = pd.read_csv(data, sep='\t')
+                    elif data_path.suffix in [".mflux"]:
+                        self.mflux = pd.read_csv(data, sep='\t', skiprows=[0])
```

Comment:
C'est beaucoup mieux comme ça! Par contre tu peux utiliser des if à la place des elif, c'est un peu plus optimisé et ça laisse la porte ouverte après à la gestion de plusieurs fichiers à la fois (les if successifs n'arrêtent pas l'éxécution du code). Aussi au lieu d'ecrire:
`if data_path.suffix in [".netw"]` 
tu peux tout simplement mettre: 
`if data_path.suffix == ".netw"`.